package.json

```diff
@@ -10,8 +10,7 @@
     "lint": "eslint src/ spec/",
     "test": "mocha --require @babel/register --exit spec/",
     "test:watch": "npm run test -- --watch",
-    "test:coverage": "nyc mocha --require @babel/register --exit spec/ && nyc report --reporter=text-lcov",
-    "test:coveralls": "npm run test:coverage | coveralls",
+    "test:coverage": "nyc mocha --require @babel/register --exit spec/ && nyc report --reporter=lcov",
```

Comment:
Doing `text-lcov` prints out the coverage report to the console, while `lcov` writes to to the `coverage` folder, so that the coveralls action can consume that output.